README.md

```diff
@@ -129,6 +129,24 @@ Pipecat makes use of WebRTC VAD by default when using a WebRTC transport layer.
 pip install pipecat-ai[silero]
 ```
 
+## Running the Krisp Audio Filter
+
+To use the Krisp Filter in this project, you’ll need access to the **Krisp C++ SDK**.
+
+### Step 1: Obtain Access to the Krisp SDK
+1. **Create a Krisp Account**: If you don’t already have an account, [sign up at Krisp](https://krisp.ai/) to access the SDK.
+2. **Download the SDK**: Once you have an account, follow the instructions on the Krisp platform to download the [Krisp's desktop SDKs](https://sdk.krisp.ai/sdk/desktop).
+3. **Export the path to you krisp SDK**:
+`export KRISP_SDK_PATH=/PATH/TO/KRISP/SDK`
+
+### Step 2: Install the `pipecat-krisp` Module
+Once the environment variable `KRISP_SDK_PATH` is exported, activate your Python virtual environment and install it with `pip`:
+
+```shell
+source venv/bin/activate
+pip install pipecat-ai[krisp]
```

Comment:
While the library has not been published yet, you can test it by creating a local distribution [using the PR](https://github.com/pipecat-ai/pipecat-krisp/pull/1) and then installing it.